xtask/src/test.rs

```diff
@@ -122,8 +123,32 @@ pub fn run_wasm() -> Result<()> {
         bail_on_err(&output, "Failed to install test dependencies")?;
     }
 
-    let output = Command::new(npm).arg("test").output()?;
-    bail_on_err(&output, &format!("Failed to run {npm} test"))?;
+    println!("Running {npm} test...");
+    let mut child = Command::new(npm)
+        .arg("test")
+        .stdout(Stdio::piped())
+        .stderr(Stdio::piped())
+        .spawn()?;
```

Comment:
Spawn inherits std{in,out,err} from the parent, so piping them and then later printing them shouldn't be necessary